RxCocoa/Foundation/URLSession+Rx.swift

```diff
@@ -7,18 +7,26 @@
 //
 
 import struct Foundation.URL
-import struct Foundation.URLRequest
 import struct Foundation.Data
 import struct Foundation.Date
 import struct Foundation.TimeInterval
-import class Foundation.HTTPURLResponse
-import class Foundation.URLSession
-import class Foundation.URLResponse
 import class Foundation.JSONSerialization
 import class Foundation.NSError
 import var Foundation.NSURLErrorCancelled
 import var Foundation.NSURLErrorDomain
 
+#if canImport(FoundationNetworking)
```

Comment:
I've been reading about this, geez. I don't know how does this affect ABI. Does this mean that if this becomes magically available on {i,mac,tv,watch}OS that it will break our ABI?

If we don't have a better workaround, it's probably fine.

Reply:
I'm really not sure, but there's no other solution right now. 
About ABI stability, @keith has a PR around this, starting with removing all of these individual imports in favour of importing all of Foundation. WDYT of that? https://github.com/ReactiveX/RxSwift/pull/2103